File: scala-advent-of-code/src/test/scala/advent/twenty_sixteen/Day03SquaresWithThreeSidesTest.scala
package advent.twenty_sixteen

import org.scalatest.{FlatSpec, Matchers}

class Day03SquaresWithThreeSidesTest  extends FlatSpec with Matchers {
  "2016 Day 03 - Input" should "calculate answers" in {
    val dp = new Day03SquaresWithThreeSides("/twenty_sixteen/Day03-SquaresWithThreeSides-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day03SquaresWithThreeSides("/twenty_sixteen/Day03-SquaresWithThreeSides-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}

File: java-advent-of-code/src/test/java/advent/twenty_sixteen/Day03SquaresWithThreeSidesTest.java
package advent.twenty_sixteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day03SquaresWithThreeSidesTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day03SquaresWithThreeSides("/twenty_sixteen/Day03-SquaresWithThreeSides-input.txt");

        assertEquals(869, dp.getPart1Answer().intValue());
        assertEquals(1544, dp.getPart2Answer().intValue());
    }
}